index.js

```diff
@@ -1,6 +1,8 @@
 var urlMapper = require('url-mapper');
-var addressbar = require('addressbar');
 var pathToRegexp = require('path-to-regexp');
+try {
+  var addressbar = require('addressbar');
+} catch(e) {}
```

Comment:
Not sure what this is for? Is it to handle failing registration of events etc. inside addressbar? Running tests?